ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -159,24 +159,37 @@ message DamlSubmitterInfo {
   string application_id = 3;
 
   reserved 4; // was maximum_record_time
-
-  // Timestamp until which the ledger MUST deduplicate this submission.
-  // Measured against the ledger's clock.
-  google.protobuf.Timestamp deduplicate_until = 5;
+  reserved 5; // was deduplicate_until
+
+  // Duration that determines the range of existing DamlLogEntries against which this
+  // submission deduplicated.
+  //  
+  // The ledger guarantees that for any existing entry e0 the deduplicate_until field of
+  // its associated DamlCommandDedupValue cdv0 and the entry e1 created from this submission
+  // it holds that 
+  //
+  //    if (e1.record_time >= e0.record_time >= e1.record_time - e1.deduplication_duration) AND
+  //       (e0.command_id, e0.application_id, e0.submitters) == 
+  //       (e1.command_id, e1.application_id, e1.submitters)
```

Comment:
Do you also want to mention the contract about not checking for application ID for a transitionary period?

Reply:
I'm inclined to not mention that as
1. It is a guarantee that I would applications to depend on only for a transitionary period of `max_deduplication_time` after an upgrade
2. It is unlikely that the potential extra rejections for the transitionary period will lead to problems or our users.

Reply:
Conclusion:
- this is an internal doc and thus we should document the guarantee that our overall impl relies on, which is the one that holds independently of the change-id

Reply:
Fixed in 8223821